lib/configuration.js

```diff
@@ -81,7 +81,12 @@ function Configuration() {
   this.conf = {
     root: this.serverRoot,
     maxLoginAttempts: 3,
-    ckEditorExtraAllowedContent: 'span(*)',
+    htmlSupportAllowed: {
+      name: ".*",
+      attributes: true,
+      classes: true,
+      styles: true
+    },
```

Comment:
Is this intentional? It looks like you've completely ignored the security section you've mentioned in the pr body.

https://ckeditor.com/docs/ckeditor5/latest/features/html/general-html-support.html#security

Reply:
I wanted to get in place a working PR, then could have a discussion with you and other developers about what would be a sensible default.

I guess we could at the minimum restrict the `script` tag?

Reply:
Generally it's best to go with a permissive, rather than a restrictive attitude to things like this. The list of potential things to restrict is always hard to determine, changes over time and tends to be more insecure as time passes.

Reply:
We'll need everything currently visible in the UI (`blockquote` `ul` `ol`) assuming they weren't working already.

Then `class` `span` `style` comes to mind. I've worked with teams who've done some weirder things, but I guess we should make it so they have to "opt in" those things themselves.

Finally my client mentioned that by default with [PR 2773](https://github.com/adaptlearning/adapt_authoring/pull/2773), that links were opening in new tabs rather than former standard behaviour of opening in the same tab.

I noticed from testing this PR that issue was fixed, so I'm wondering if the attribute `target` was getting removed. If so, we should also add it to the whitelist.

Reply:
I have a minor update to support these things, would you mind if i commit them?

Reply:
Gor for it!

Reply:
I think, there's probably an issue with restricting attribute values with the way I've done this.